conf/broker.conf

```diff
@@ -35,6 +35,11 @@ brokerServicePort=6650
 # Broker data port for TLS - By default TLS is disabled
 brokerServicePortTls=
 
+# Specify the ssl provider for the broker service:
+# When using the CACert, available values are one of OpenSSL and JDK.
```

Comment:
```suggestion
# When using TLS authentication with CACert, the valid value is either OpenSSL or JDK.
```

Reply:
Thanks @momo-jun, should be resolved now.